curve25519/Cargo.toml

```diff
@@ -15,12 +15,16 @@ libcrux-hacl-rs = { version = "=0.0.3", path = "../hacl-rs/" }
 libcrux-macros = { version = "=0.0.3", path = "../macros" }
 libcrux-traits = { version = "=0.0.3", path = "../traits" }
 libcrux-secrets = { version = "=0.0.3", path = "../secrets"}
+libcrux-hkdf = { version = "=0.0.3", path = "../libcrux-hkdf", optional = true }
 
 [features]
+kem-api = ["libcrux-hkdf"]
```

Comment:
Do we have naming convention for these? It could also just be `kem`. I'm fine either way.